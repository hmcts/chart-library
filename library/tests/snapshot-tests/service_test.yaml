suite: Snapshot tests
templates:
  - templates/v2/service.yaml
tests:
  - it: manifest should match snapshot
    asserts:
      - matchSnapshot: {}