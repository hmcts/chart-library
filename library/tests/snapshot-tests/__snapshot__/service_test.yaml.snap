manifest should match snapshot:
  1: |
    apiVersion: v1
    kind: Service
    metadata:
      labels:
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: release-name-library
        helm.sh/chart: library-0.1.0
      name: release-name-library
    spec:
      ports:
        - name: http
          port: 80
          protocol: TCP
          targetPort: 3000
      selector:
        app.kubernetes.io/name: release-name-library
